frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -474,11 +474,33 @@ object Project {
       options ++ ysemanticdb ++ sourceRoot
     }
 
-    def enableJavaSemanticdbOptions(options: List[String]): List[String] = {
-      if (hasJavaSemanticDBEnabledInCompilerOptions(options)) options
+    def hasProcessorPath(options: List[String]): Boolean =
+      options.contains("-processorpath")
+
+    def includeSemanticdbInProcessorPath(
+        options: List[String],
+        pluginPath: AbsolutePath
+    ): List[String] = {
+      if (hasProcessorPath(options)) {
+        val (before, after) = options.splitAt(options.indexOf("-processorpath") + 1)
+        if (!after.head.contains(pluginPath)) {
+          before ::: s"${after.head}:$pluginPath" :: after.tail
+        } else options
+      } else options
+
+    }
+
+    def enableJavaSemanticdbOptions(
+        options: List[String],
+        pluginPath: AbsolutePath
+    ): List[String] = {
+      val pluginOptions = includeSemnaticDBInProcessorPath(options, pluginPath)
```

Comment:
```suggestion
      val pluginOptions = includeSemanticdbInProcessorPath(options, pluginPath)
```

Did you rename via Metals and it failed? :O

Reply:
no I applied the suggestion directly in gh 😮‍💨 
sorry for the noise